### ISSUE TITLE
periorbital region

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#2285]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
abnormal periorbital region morphology

>2. Synonyms (e.g., absent spleen)
abnormal periorbital area morphology; abnormal periorbital region; periorbital region malformation

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
DEF: any structural anomaly of the region around the eye socket
HP:0000606 
>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
parent term: abnormal facial morphology MP:0003743 

NOTE: existing term periorbital edema MP:0001788 should be made a child of abnormal periorbital region morphology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

Anna

**Comment by anna-anagnostop** [comment]
Added abnormal periorbital region morphology MP:0030038 and made periorbital edema MP:0001788 a child term.
